src/main/java/ca/bc/gov/backendstartapi/util/ObjectUtil.java

```diff
@@ -8,15 +8,11 @@
 /** This class contains utils methods. */
 public class ObjectUtil {
```

Comment:
For Utils classes, always create a private constructor or annotate it with `@NoArgsConstructor(access=PRIVATE)` to prevent instantiation

Reply:
Right, so this is why it was there. I completely forgot about this.